sim/visuals/microbit.ts

```diff
@@ -307,14 +310,18 @@ path.sim-board {
         private rssi: SVGTextElement;
         private lightLevelButton: SVGCircleElement;
         private lightLevelGradient: SVGLinearGradientElement;
+        private lightLevelInittialized = false;
```

Comment:
Typo, but change needs to be applied to usage elsewhere

```suggestion
        private lightLevelInitialized = false;
```